app/views/ppm_estimator/_estimate_table.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="ppm-estimate-table-title">
+  Your PPM Incentive Estimate:<a id="ppm-edit-details" onclick="onEditDetails()" href="javascript:void(0);">Edit Details</a>
+</div>
+<div class="ppm-estimate-table">
+  Estimated Transportation Incentive<br>
+  <span class="ppm-estimate-amount">$<%= @ppm_estimator.incentive_without_packing_range.min %>–$<%= @ppm_estimator.incentive_without_packing_range.max %></span>
+  <span class="ppm-estimate-detail">
+    From <%= @ppm_estimator.start_zip3.basepoint_city %>, <%= @ppm_estimator.start_zip3.state %> <%= params['start'] %>
+    to <%= @ppm_estimator.end_zip3.basepoint_city %>, <%= @ppm_estimator.end_zip3.state %> <%= params['end'] %>
+  </span><br>
+  <span class="ppm-estimate-detail">
+    Departing <%= @ppm_estimator.date.strftime('%B %-d, %Y') %><% if @ppm_estimator.peak_season? %> (Peak Season)<% end %><br>
+    <%= @ppm_estimator.full_weight %> lbs
+  </span>
+</div>
+<div class="ppm-estimate-table">
+  Estimated Packing Incentive<br>
+  <span class="ppm-estimate-detail">
+    <%= @ppm_estimator.packing_weight %>
+  </span>
+  <span class="ppm-estimate-amount">
+    + <%= @ppm_estimator.packing_incentive %>
+  </span>
+</div>
+<div class="ppm-estimate-table-total">
+  <span class="ppm-estimate-detail-total">Total Estimated Incentive</span>
+  <span class="ppm-estimate-amount-total">$<%= @ppm_estimator.total_incentive_range.min %>–$<%= @ppm_estimator.total_incentive_range.max %></span>
+  Maximum Advance Payment
+  <span class="ppm-estimate-amount">$<%= @ppm_estimator.advance_range.min %>–$<%= @ppm_estimator.advance_range.max %> (<%= @ppm_estimator.advance_percentage %>%)</span>
+</div>
+
+<%= link_to ppm_estimator_path do %>
+  <button class="usa-button-outline">Start a New Estimate</button>
+<% end %>
```

Comment:
Is this UI element an anchor or a button? It should be one or the other, not both.